doc/chapters/integration-guidelines/10-traffic-shaping.rst

```diff
@@ -0,0 +1,66 @@
+Traffic Shaping
+===============
+
+The Linux offers a very rich set of tools for managing and manipulating the transmission of packets.
+Those tools can be used to manage and shape traffic on SoftwareContainer.
+
+The Linux network routing, firewalling and traffic control subsystem is very powerful and flexible
+and have grown to be a very mature stack of tools. To glance those solutions the linux
+documentation project would be very good resource to start. Please check
+`TLDP <http://www.tldp.org/HOWTO/html_single/Traffic-Control-HOWTO/#intro>`_
+for having understanding of the concept. `Linux Advanced Routing & Traffic Control <http://lartc.org/howto/>`_
```

Comment:
s/for having understanding of the concept/for documentation on the concepts of traffic control in Linux.